src/friktion/ids.ts

```diff
@@ -1,5 +1,16 @@
 import { PublicKey } from "@solana/web3.js";
 
-export const VOLT_PROGRAM_ID = new PublicKey("VoLT1mJz1sbnxwq5Fv2SXjdVDgPXrb9tJyC8WpMDkSp");
+export const VOLT_PROGRAM_ID = new PublicKey(
+  "VoLT1mJz1sbnxwq5Fv2SXjdVDgPXrb9tJyC8WpMDkSp"
+);
 
-export const VOLT_FEE_OWNER = new PublicKey("3KjJiWBfaw96qGhysq6Fc9FTxdPgPTNY6shM7Bwfp8EJ");
+export const VOLT_FEE_OWNER = new PublicKey(
+  "3KjJiWBfaw96qGhysq6Fc9FTxdPgPTNY6shM7Bwfp8EJ"
+);
+
+export const IS_STABLE = [
```

Comment:
What do these keys represent? Perhaps need a more descriptive name instead of `IS_STABLE`